lib/rmg/CRmgTemplateZone.cpp

```diff
@@ -920,6 +1424,7 @@ bool CRmgTemplateZone::createTreasurePile(int3 &pos, float minDistance, const CT
 		else
 		{
 			object = oi.generateObject();
+			object->appearance = oi.templ;
```

Comment:
Actual fix for seer hut's problem

Reply:
Awesome!

It might fix some other cases, like Dragon Fly Hive.